packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3736,6 +3738,41 @@ export interface SpokenLanguageStrings {
     'zh-tw': string;
 }
 
+// @public
+export const StarSurvey: (props: StarSurveyProps) => JSX.Element;
+
+// @public
+export interface StarSurveyProps {
+    onSubmitStarSurvey: (ratings: number, type: StarSurveyTypes) => Promise<void>;
+    selectedIcon?: string;
+    showSurvey?: boolean;
+    strings?: StarSurveyStrings;
+    styles?: StarSurveyStyles;
+    type?: StarSurveyTypes;
```

Comment:
depluralize `StarSurveyTypes` -> `StarSurveyType` or `StarSurveyKind` (reason being the variable is of a single kind so although the type spans multiple kinds the typename should be what the variable will be)